README.md

```diff
@@ -0,0 +1,21 @@
+# Shopping List
+This is vanilla JavaScript project with Shopping List. You can type product name, choose its category and click button to add product to list. You can check product as done/bought and after that checked product goes to the end of list with additional style. You can also uncheck product and it goes as last in unchecked part. New added products also go as last in unchecked part, before checked positions. List is saved in local storage.
+
+## Features
+- adding products to list with choosing product category
+- sorting products as checked and unchecked
+- removing products from list
+- one-click quick adding with several products
+- saving in local storage
+
+## Features to add
+- sorting by category name within unchecked products
+- English version
+- choosing recipe and adding necessary products to list 
+
+## Further improvements
+- improve naming
+- extract reusable functions
+- change functions to pure when it's possible
+- add comments with information what particular functions do
+- add futher changes in parts, in separeted branches and commits
```

Comment:
I would rather omit this comment because this is about good practice in using git. I would write only about code improvements.